Improved the Choosing a shader and Shaders and Materials sections. Also OpenGL in Deferred. (#6869)
Unity-Technologies/Graphics: com.unity.render-pipelines.universal/Documentation~/shaders-in-universalrp.md

- Autodesk Interactive Transparent
- Autodesk Interactive Masked

## Shader compatibility

Lit and custom Lit shaders written for the Built-in Render Pipeline are not compatible with URP.

Unlit shaders written for the Built-in Render Pipeline are compatible with URP.

For information on converting shaders written for the Built-in Render Pipeline to URP shaders, see the page [Converting your shaders](upgrading-your-shaders.md).

## Choosing a shader

The Universal Render Pipeline implements Physically Based Rendering (PBR).

The pipeline provides pre-built shaders that can simulate real world materials.

> When using PBR shaders, Materials behave consistently under different lighting conditions. PBR materials provide a set of parameters that let artists achieve consistency across hardware in Scenes with different material types.

Reviewer: ```suggestion
PBR materials provide a set of parameters that let artists achieve consistency between different material types and under different lighting conditions.
```
Reworded the sentence to remove **hardware** consistency, that is not achieved by PBR. I can try to explain a little better what I mean and you can also reword the sentence to something better. You can think of PBR as a framework that provides some parameters that model materials, these parameters are physically based. When all materials in the scene use PBR, it's easier to guarantee that they look correctly when comparing materials in a scene with each other (metals, rubber, wood, glass, car paint, hair etc) and when lighting conditions change. 

If a project doesn't use PBR it means that it relies much more on artistic direction to unify the look of these different material types.
